src/pycram/costmaps.py

```diff
@@ -5,7 +5,7 @@
 from pycram.robot_description import InitializedRobotDescription as robot_description
 from .bullet_world import BulletWorld
 from .bullet_world_reasoning import _get_images_for_target
-from .ik import request_ik, IKError
+from .external_interfaces.ik import request_ik, IKError
```

Comment:
Is this still used ? If not it should be removed

Reply:
Will be done in a later PR

Reply:
no but this will be in one PR with all the other imports